src/components/Reservations/index.js

```diff
@@ -0,0 +1,43 @@
+import { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { getMyReservations } from '../../redux/actions/reservationActions';
+
+export default function Reservations() {
```

Comment:
- [ ] When I try to add a reservation:

Step 1:

![image](https://github.com/c00p75/appointment-app-frontend/assets/57421823/3e278a27-5916-40f7-a234-6f0c515bbfcf)

The app immediately sends me to reservations, but this happens:

![image](https://github.com/c00p75/appointment-app-frontend/assets/57421823/adb25a33-8d1f-43d1-a65a-798237df5777)

Afterward, If I reload the page, the issue is gone:

![image](https://github.com/c00p75/appointment-app-frontend/assets/57421823/003b4303-3420-4d0e-9d34-f9b54629f135)

Kindly, fix this bug so your app works as intended 👍

Reply:
fixed, thanks!